src/V3Inline.cpp

```diff
@@ -385,6 +326,7 @@ class InlineRelinkVisitor final : public VNVisitor {
     void visit(AstNodeAssign* nodep) override {
         if (const AstVarRef* const varrefp = VN_CAST(nodep->lhsp(), VarRef)) {
             if (m_initialStatic && varrefp->varp()->user2() && varrefp->varp()->user4()) {
+                // TODO: WHYYYYYY?????????????????
```

Comment:
Put a FIXME so comments like this fail until resolved please.

I think this is because if e.g.

input a = 0;

if unconnected the initial matters, but if connected to an upper module with .a(upper_a),  then upper_a is a continuous assignment that will always override the submodule's a.

Reply:
Thanks, I guess my real question was why only for variables that don't use the aliases. I will look into this further.

Reply:
Actually this is a rabbit hole I'm not going to go down in right now. There is an implicit initializer to 0 (or x in 4-state), so strictly speaking we wouldn't be able to use aliases for anything if we want to make the above always work, so I won't for now. 

The cleanup of the static initializers where this comment was added is actually taken care of by the insertion of placeholder variables that I added to fix the malformed code as well, so no need to handle them explicitly any more, will delete.